drivers/gles2/shaders/canvas.glsl

```diff
@@ -701,5 +701,13 @@ FRAGMENT_SHADER_CODE
 //use lighting
 #endif
 
+#ifdef LINEAR_TO_SRGB
+	// regular Linear -> SRGB conversion
+	float a = 0.055;
+	color.r = color.r < 0.0031308 ? (12.92 * color.r) : ((1.0 + a) * pow(color.r, 1.0 / 2.4) - a);
+	color.g = color.g < 0.0031308 ? (12.92 * color.g) : ((1.0 + a) * pow(color.g, 1.0 / 2.4) - a);
+	color.b = color.b < 0.0031308 ? (12.92 * color.b) : ((1.0 + a) * pow(color.b, 1.0 / 2.4) - a);
+#endif
```

Comment:
For some reason for the GLES2 renderer on desktop to work, I had to write these out....

Reply:
From the GLSL 120 spec:

> The relational operators greater than (>), less than (<), greater than or equal (>=), and less than or
equal (<=) operate only on scalar integer and scalar floating-point expressions. The result is scalar
Boolean. Either the operands’ types must match, or the conversions from Section 4.1.10 “Implicit
Conversions” will be applied to the integer operand, after which the types must match. To do
component-wise relational comparisons on vectors, use the built-in functions lessThan,
lessThanEqual, greaterThan, and greaterThanEqual.

That being said, you can still go back to the original mix implementation by constructing a ``vec3`` out of the ``bvec3``. E.g. instead of ``mix(a, b, lessThan())``, do ``mix(a, b, vec3(lessThan()))``

Reply:
I think that is worth changing